### ISSUE TITLE
Is the project still alive?

### ISSUE THREAD

**Issue by hpe-ykoehler** [disroop/DisroopEmbeddedHipster#30]
Hi,

I have built something very similar, I am not as advance as this project implies to be, but I was able to get to a working point with devcontainer, conan 2.0.0, cmake, gcc, and unit test (using unity/fff), etc.

Having found this project, I could see how I could try to merge my stuff with what is found in here to get forward but I am unable to see some of the features presented in here works due to build error and such.

So just wanted to know if this project was still alive or not.

**Comment by stefaneicher** [comment]
hi thanks for your message! The projects is a bit at sleep because we do not find time to work on it. But we plan to continue. We have a lot of stuff we could add. In case you are interested we can have a chat about it. what would you use the project fore? you can also contact me under stefan.eicher@disroop.ch

**Comment by hpe-ykoehler** [comment]
I am reworking an old C build system using Conan 2.0.0 and C, I ended up with pretty much the same results as here.  But using conan2, and CMake Presets let me rethink some aspects in a different way.  Saw the cmake-init project also which contains interesting presets that I integrated.

Eventually, I was hoping to contribute back my stuff, and then I saw your project, so eventually, I may push to it some of the stuff I used.

**Comment by stefaneicher** [comment]
Yes let us know what interesting ideas you have. I fulltime work on such issues just not in thus opensource project
